### PR TITLE
pkp/pkp-lib#9627 Add indexes to usage stats temporary tables 

### PR DESCRIPTION
**Pull request by bozana**
s. https://github.com/pkp/pkp-lib/issues/9627

### CODE REVIEW

File: dbscripts/xml/upgrade.xml [hunk 1]
```diff
@@ -215,6 +215,7 @@
 	<upgrade minversion="3.1.0.0" maxversion="3.4.0.9">
```
Comment (jonasraoni) [inline dbscripts/xml/upgrade.xml:215]: Hmm, the latest release is `3.4.0.4`, and these migrations look like extra fixes for the `3.4`. Therefore, shouldn't the `maxversion` here be `3.4.0.3`?

Comment (jonasraoni) [reply dbscripts/xml/upgrade.xml:215]: I see it this way:
- User upgrades to `3.4.0.4`
- These migrations will be executed
- User upgrades to `3.4.0.5` (next release), the current `maxversion` will still fit, and the migrations will probably be executed again (hopefully they can be re-executed without errors).

Comment (jonasraoni) [reply dbscripts/xml/upgrade.xml:215]: Following the same line of thought I think this new migration should be placed on another `<upgrade>` entry, perhaps: `<upgrade minversion="3.1.0.0" maxversion="3.4.0.4">`

Comment (jonasraoni) [reply dbscripts/xml/upgrade.xml:215]: As it's late here, tagging @asmecher to confirm 🤪

Comment (bozana) [reply dbscripts/xml/upgrade.xml:215]: @asmecher said once -- if you remember we already have this discussion :-) -- that the user do not execute DB upgrade for minor versions, that's why we always consider the 9 at the end...

Comment (jonasraoni) [reply dbscripts/xml/upgrade.xml:215]: I just did a test (upgraded my 3.4.0.4 to a fake 3.4.0.5) and the migrations were re-executed, so I think it's better to fix.

Comment (bozana) [reply dbscripts/xml/upgrade.xml:215]: I will let @asmecher decide. 
From this upgrade group:
- I9136_MigrateUniqueSiteId was introduce in 3.4.0.1
- I9231_FixMetricsIndexes in 3.4.0.2
- I9627_AddUsageStatsTemporaryTablesIndexes (this one) will be in 3.4.0.5/3.4.1
It seems that all of them can be executed several times without problems, but...

**Comment by bozana** [comment]
@jonasraoni, the second commit here is where I decrease the column IP to 64 characters.
If OK I will merge it all into one commit...

**Review by jonasraoni — APPROVED** [review]
I approved! There's just the `upgrade.xml` issue remaining, but I think it can be handled by a separated commit.

File: dbscripts/xml/upgrade.xml (hunk 1 above)
Comment (jonasraoni) [reply dbscripts/xml/upgrade.xml:215]: I think this can be moved to another issue as well.

Comment (jonasraoni) [reply dbscripts/xml/upgrade.xml:215]: I've created this issue: https://github.com/pkp/pkp-lib/issues/9637 and added a milestone for 3.4.0-5 to not forget about it, so I think you can merge this :)

**Pull request merged**